ee/clickhouse/models/cohort.py

```diff
@@ -307,13 +283,13 @@ def recalculate_cohortpeople(cohort: Cohort) -> Optional[int]:
         size_before=before_count[0][0],
     )
 
-    cohort_filter = GET_PERSON_IDS_BY_FILTER.format(
-        distinct_query="AND " + cohort_filter,
-        query="",
-        offset="",
-        limit="",
-        GET_TEAM_PERSON_DISTINCT_IDS=get_team_distinct_ids_query(cohort.team_id),
-    )
+    # cohort_filter = GET_PERSON_IDS_BY_FILTER.format(
```

Comment:
@EDsCODE do you know if there was a reason why this specific bit for inserts/deletes was so convoluted?

I might be missing something with this simplification

Reply:
Ahhh 🤦